### PR TITLE
Output schema theme in dataLayer

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This adds an identifying key:value in the `dataLayer[ ]` array to allow Google Analytics to segment events by. There will be scenarios where it is not included at all (e.g. 404 where a user hasn't had an authenticated session). This has been agreed with the analytics engineers.

### How to review 
Change if this new `dataLayer[]` is generated correctly in head element of each page.

### Checklist

* [x] New static content marked up for translation
* [x] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by petechd** [comment]
[Output NISRA or Non-NISRA in dataLayer array (S)](https://trello.com/c/w6ZtGKNl/4087-output-nisra-or-non-nisra-in-datalayer-array-s)

**Comment by petechd** [comment]
[Output NISRA or Non-NISRA in dataLayer array (S)](https://trello.com/c/w6ZtGKNl/4087-output-nisra-or-non-nisra-in-datalayer-array-s)

File: app/helpers/template_helper.py [hunk 1]
```diff
@@ -120,3 +121,13 @@ def safe_content(content):
...
+
+
+def get_data_layer(schema_theme):
+    if schema_theme == "census-nisra":
+        return "dataLayer = [{nisra : true}];"
+
+    if schema_theme == "census":
+        return "dataLayer = [{nisra : false}];"
+
+    return "dataLayer = [];"
```
Comment (MebinAbraham) [inline app/helpers/template_helper.py:133]: I think we should return a list instance and not a string, so:
```suggestion
    if schema_theme == "census-nisra":
        return [{"nisra": True}]

    if schema_theme == "census":
        return [{"nisra": False}]

    return []
```
Then in `_base.html` you can do:
```jinja
	<script nonce="{{ csp_nonce() }}">
      dataLayer = {{ data_layer | tojson }}
    </script>
```

Comment (petechd) [reply app/helpers/template_helper.py:133]: But do we want nisra key as a string? Proposed output is `[{nisra : [true | false] }]`..

Comment (MebinAbraham) [reply app/helpers/template_helper.py:133]: It is still a valid object. `{"nisra": false}` and `{nisra: false}` are identical in JS.

Comment (petechd) [reply app/helpers/template_helper.py]: Done! thanks!

**Review by berroar — APPROVED** [review]

File: app/helpers/template_helper.py [hunk 2]
```diff
@@ -78,6 +78,7 @@ def render_template(template, **kwargs):
         language_code=get_locale().language,
         survey_title=cookie_session.get("survey_title"),
         cdn_url=cdn_url,
+        data_layer=get_data_layer(cookie_session.get("theme")),
```
Comment (JamesGardiner) [inline app/helpers/template_helper.py:81]: It's probably worth defining a theme variable here, given the number of times the cookie_session is accessed

**Review by MebinAbraham — APPROVED** [review]
The semicolon is now missing from the JS statement but shouldn't matter as it is still valid.

**Comment by petechd** [comment]
> The semicolon is now missing from the JS statement but shouldn't matter as it is still valid.

As far as I know, semicolon in JS is only optional - good practices, indicates end of a statement, but that's all.

File: app/helpers/template_helper.py (hunk 2 above)
Comment (petechd) [reply app/helpers/template_helper.py]: Done, thanks!

**Review by MebinAbraham — APPROVED** [review]

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**